src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="first">
+          <label for="fname">Surname:</label>
+          <input
+        type="text"
+        id="fname"
+        name="fname"
+        autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            id="fname"
            name="fname"
            autocomplete="off"
          >
```
correct formatting :arrow_up: 
please, check everywhere